### ISSUE TITLE
[FEATURE] More options for Lift-Calling

### ISSUE THREAD

**Issue by xCraimax** [SBDPlugins/V10Lift#81]
**Is your feature request related to a problem? Please describe.**
/

**Describe the solution you'd like**
Being able to specify a button, which send the elevator up/down by one floor.
Being able to specify a lever, which sends the elevator up/down (going from the enabled/disabled state of the lever), when there's only two floors.

**Describe alternatives you've considered**
/

**Additional context**
/

**Comment by stijnb1234** [comment]
Why would you need a lever for this? Never seen an elevator with a lever.

**Comment by xCraimax** [comment]
I've never seen an elevator with a lever aswell.
The lever was an idea I got, but a button is fine aswell.

**Comment by stijnb1234** [comment]
I agree there should be a button to go to a specific floor directly. Should it add the action to the queue, or go directly to that floor? The problem is that the user can also queue floors inside the cab.

**Comment by xCraimax** [comment]
As I think, the queue from inside the cab should be prioritized.
